# Used to publish the state of a controller.
# If control is not active (active=false) all other values will be 0.
bool active
int16 sensor
int16 target
int16 p
int16 i
int16 d
